dense-numericals-src/basic-math/minimum.lisp

```diff
@@ -144,22 +144,18 @@
                                   &key ((axes null))
                                   ((out null))
                                   ((keep-dims null)))
-    real
+    <type>
```

Comment:
I'm not totally sure how the "parametric types" behave outside the [orthogonally specializing types](https://gitlab.com/digikar/peltadot/-/blob/main/docs/ortho-types-polymorphic-functions.org). I will revisit this again.

Reply:
https://github.com/digikar99/numericals/pull/20#issuecomment-2628783169